gaphor/diagram/tools/shortcut.py

```diff
@@ -32,3 +35,20 @@ def delete_selected_items(view: GtkView, event_manager):
             if i.subject and i.subject in self_and_owners(i.diagram):
                 del i.diagram.element
             i.unlink()
+
+
+def unselect(view: GtkView, event_manager):
```

Comment:
I would use a more descriptive name here:

```suggestion
def cancel_handle_drag(view: GtkView, event_manager):
```